constants/network.ts

```diff
@@ -19,10 +19,12 @@ export enum BlastNetwork {
 	ETHEREUM_MAINNET = 'eth-mainnet',
 	OPTIMISM_MAINNET = 'optimism-mainnet',
 	ETHEREUM_KOVAN = 'eth-kovan',
```

Comment:
Suggest to switch to L1 Goerli here too and remove Kovan support once and for all